### DISCUSSION TITLE
Delay in receiving Student Developer Pack benefits

### DISCUSSION THREAD

**Discussion by haddag-n** [community/community#140094] [GitHub Education]
### Hi everyone,

Hello,

I was approved for the GitHub Student Developer Pack on September 23rd, and I was informed that the educational benefits would be available within 3 to 5 days. However, today is September 29th, more than 5 days after approval, and I still have not received my benefits.

![Capture d’écran 2024-09-29 à 10 20 29](https://github.com/user-attachments/assets/bec5f0b7-128b-4573-a99d-9558374fdf7d)

Since the beginning of the university semester is imminent, it is crucial for me to have access to the pack in order to start my courses and projects effectively. Please find attached a screenshot of my approved status.

Thank you in advance for your assistance and understanding.

Best regards,

**Comment by davevad93** [comment]
Hi @digitup-nhaddag ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @digitup-nhaddag was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
